runtime/testy/runtime.py

```diff
@@ -95,6 +95,8 @@ def nonTestModePrint(*args):
         print(newBadThing.event)
         nonTestModePrint(newBadThing.data)
         if newBadThing.event in restartEvents:
+          if newBadThing.event in studentErrorEvents:
+            stateQueue.put([SM_COMMANDS.SEND_CONSOLE, [newBadThing.stackTrace]])
```

Comment:
I dont think this contains the actual error. (newBadThing.event)

you can just pass in newBadThing as an argument though, instead of newBadThing.stacktrace.
newBadThing objects have a built in print method that does this already i believe.